package.json

```diff
@@ -54,7 +46,8 @@
     "graphql": "^15.5.2",
     "graphql-tag-ts": "^0.1.16",
     "lodash": "^4.17.21",
-    "nodemailer": "^6.6.3",
```

Comment:
why is this removed?

Reply:
They were imports that weren't being used

Reply:
we don't use `nodemailer`? why was this added originally? you were trying stuff for email stuff and forgot to remove?

Reply:
Yeah, I forgot to remove at the time. I was trying to remove all the packages that I didn't end up using up until now and I didn't see it anywhere being used and I didn't use it in my sendemail.ts